spec/rescue_unique_constraint_spec.rb

```diff
@@ -2,31 +2,57 @@
 require 'rescue_unique_constraint'
 
 describe RescueUniqueConstraint do
-  before do
+  before :all do
     ActiveRecord::Base.establish_connection(:adapter => "sqlite3", :database => ":memory:")
     ActiveRecord::Schema.verbose = false
     ActiveRecord::Schema.define(:version => 1) do
       create_table :things do |t|
         t.string :name
         t.string :test
+        t.integer :code
+        t.integer :score
       end
 
       add_index :things, :name, unique: true, name: "idx_things_on_name_unique"
       add_index :things, :test, unique: true, name: "idx_things_on_test_unique"
+      add_index :things, [:code, :score], unique: true, name: "idx_things_on_code_and_score_unique"
     end
   end
 
   class Thing < ActiveRecord::Base
     include RescueUniqueConstraint
     rescue_unique_constraint index: "idx_things_on_name_unique", field: "name"
     rescue_unique_constraint index: "idx_things_on_test_unique", field: "test"
+    rescue_unique_constraint index: "idx_things_on_code_and_score_unique", field: "score"
+  end
+
+  before :each do
+    Thing.destroy_all
   end
 
   it "rescues unique constraint violations as activerecord errors" do
-    thing = Thing.create(name: "foo", test: 'bar')
-    dupe = Thing.new(name: "foo", test: 'bar')
+    thing = Thing.create(name: "foo", test: 'bar', code: 123, score: 1000)
+    dupe = Thing.new(name: "foo", test: 'baz', code: 456, score: 2000)
+    expect(dupe.save).to eql false
+    expect(dupe.errors.messages).to have_key(:name)
```

Comment:
I noticed that all these tests still pass with the old code. If you change this to a more specific assertion:

```ruby
expect(dupe.errors.messages.key).to eql([:name])
```

then I think that better expresses the intent of this PR.